rootfs/etc/ssh/sshd_config

```diff
@@ -26,3 +26,6 @@ TCPKeepAlive yes
 #AcceptEnv LANG LC_*
 Subsystem sftp /usr/lib/openssh/sftp-server
 UseDNS no
+Ciphers aes128-cbc,aes192-cbc,aes256-cbc,rijndael-cbc@lysator.liu.se,aes128-ctr,aes192-ctr,aes256-ctr,aes128-gcm@openssh.com,aes256-gcm@openssh.com,chacha20-poly1305@openssh.com
+Kexalgorithms diffie-hellman-group-exchange-sha256,ecdh-sha2-nistp256,ecdh-sha2-nistp384,ecdh-sha2-nistp521,curve25519-sha256@libssh.org,gss-gex-sha1-,gss-group1-sha1-,gss-group14-sha1-
```

Comment:
Great PR @pdomagala ! Thank you for looking into this and opening this PR. Security PRs are very much appreciated because they improve the project for all users. @kingdonb is right, we should update openssh/ssh version and configs inside of the deis base image as well.

One thing that I noticed after doing some reading on the `ssh_config` file is that the convention is to list the Ciphers from greater security (confidentiality + integrity) to lesser security. Could you list the Kexalgorithms in the following order, not sure if order matters here?

```
KexAlgorithms curve25519-sha256@libssh.org,ecdh-sha2-nistp521,ecdh-sha2-nistp384,ecdh-sha2-nistp256,diffie-hellman-group-exchange-sha256
```

We should list all of sections them in terms greater to weaker security. 

I think curve25519 elliptical cryptography is a bit more secure (but only in terms of forward secrecy and collisions) than diffie-hellman key exchange but I'm not an expert in this.

For example, for the Ciphers, it should be this:

```
Ciphers chacha20-poly1305@openssh.com,aes256-gcm@openssh.com,aes128-gcm@openssh.com,aes256-ctr,aes192-ctr,aes128-ctr
```

Some more information for why aes256-GCM vs CBC vs CTR vs ECB for the Ciphers:

https://crypto.stackexchange.com/a/43289
https://security.stackexchange.com/questions/184305/why-would-i-ever-use-aes-256-cbc-if-aes-256-gcm-is-more-secure

> AES-GCM is a more secure cipher than AES-CBC, because AES-CBC, operates by XOR'ing (eXclusive OR) each block with the previous block and cannot be written in parallel.

> As for any advantages it might have, CBC doesn't fail as catastrophically if the IV is reused, and it can be faster if implemented on basic hardware.

> As for GCM, it's basically GCM = CTR + Authentication (not CBC). It's fast and secure if used correctly, and very versatile, hence its popularity.

> CBC is an older mode that was invented back in the dark ages cryptographically speaking (no later than the 1970s), and is now disfavored because of the lack of built-in authentication and all the trouble that's been caused by padding oracles. One good practical example of this is that TLS 1.3 got rid of support for CBC.

I think the MACs are listed in the correct order, so great job there! :+1: 

Probably more information you wanted in a PR review, but we are learning together. :)

Reply:
>ssh(1)/sshd(8): add a KexAlgorithms knob to the client and server
configuration to allow selection of which key exchange methods are
used by ssh(1) and sshd(8) and their order of preference.

Good catch! I'll fix it in a minute.

Reply:
@Cryptophobia Okay, I sorted it like you said, please just verify.